Lesson 5

NatalkaBal opened the pull request:
1. Вынести поиск в отдельный компонент.
2. Вынести корзину в отдельный компонент.

NatalkaBal committed:
lesson-4

1. Добавить методы и обработчики событий для поля поиска. Создать в объекте данных поле searchLine и привязать к нему содержимое поля ввода. На кнопку «Искать» добавить обработчик клика, вызывающий метод FilterGoods.
2. Добавить корзину. В html-шаблон добавить разметку корзины. Добавить в объект данных поле isVisibleCart, управляющее видимостью корзины.
3. *Добавлять в .goods-list заглушку с текстом «Нет данных» в случае, если список товаров пуст.

NatalkaBal committed:
lesson-5

lesson-5

NatalkaBal committed:
lesson-5-1

Tardigrada777 reviewed (approved):
На самом деле неплохо. Мы в начале следующего урока обсудим еще одну тему, и тогда Вы сможете довести код до ума, организовав полноценное общение между компонентами.

NatalkaBal committed:
Update .gitignore

NatalkaBal committed:
Update .gitignore

NatalkaBal committed:
lesson-7

1. Привязать добавление товара в корзину к реальному API.
2. Добавить API для удаления товара из корзины.

NatalkaBal committed:
lesson-6-1

lesson-6
